src/encoded/static/scss/encoded/modules/_common_item.scss

```diff
@@ -15,3 +15,38 @@
         flex: 0 1 auto;
     }
 }
+
+// doi: (Digital Object Identifier) references, including Copy button.
+$doi-ref-border-color: #a0a0a0;
```

Comment:
I did the square corners deliberately to avoid people thinking the whole thing was a button. I originally had no border around the doi text, but that made the button looks weird to me, standing on its own after the text. Putting the button to the left of the doi text looked better but it felt strange to have the Copy button before the text it refers to. And I couldn’t make the whole thing a button because people might want to copy the doi text. Any other suggestions? I didn’t know what else to do, and this seemed not that confusing to me.

I could add more space below the title. I minimized the space because I thought it likely Ben would want less space.